pkg/dataprotection/backup/deleter.go

```diff
@@ -235,7 +235,9 @@ func (d *Deleter) DeleteVolumeSnapshots(backup *dpv1alpha1.Backup) error {
 
 	snaps := &vsv1.VolumeSnapshotList{}
 	if err := vsCli.List(snaps, client.InNamespace(backup.Namespace),
-		client.MatchingLabels(BuildBackupWorkloadLabels(backup))); err != nil {
+		client.MatchingLabels(map[string]string{
+			dptypes.DataProtectionLabelBackupNameKey: backup.Name,
```

Comment:
`dptypes.DataProtectionLabelBackupNameKey` renamed in main branch, merge the main branch.